### ISSUE TITLE
Invalid block tag: 'trans' on admin.html

### ISSUE THREAD

**Issue by avara1986** [tomchristie/django-rest-framework#3575]
**TemplateSyntaxError: Invalid block tag: 'trans', expected 'elif', 'else' o 'endif** on rest_framework/templates/rest_framework/admin.html line 117:
```
114	                {% if filter_form %}
115	                    <button style="float: right; margin-right: 10px" data-toggle="modal" data-target="#filtersModal" class="btn btn-default">
116	                        <span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
117	                        {% trans "Filters" %}
118	                    </button>
119	                {% endif %}
```
If you add this code at line 1 this issue is solved:
```
{% load i18n %}
```

**Comment by xordoquy** [comment]
Duplicate of #3564

**Comment by jpadilla** [comment]
Resolved in #3564
